app/uk/gov/hmrc/debttransformationstub/controllers/ETMPController.scala

```diff
@@ -69,6 +69,23 @@ class ETMPController @Inject()(environment: Environment, cc: ControllerComponent
 
   }
 
+  def paymentPlanEligibility(
+    regimeType: String,
+    idType: String,
+    idValue: String,
+  ): Action[AnyContent] = Action { request =>
+    val queryKeys: List[String] =
```

Comment:
I think add a TODO to show that these lines will be used in future if other files are needed